token/cli/src/main.rs

```diff
@@ -4459,6 +4651,109 @@ fn app<'a, 'b>(
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::ConfigureConfidentialTransferAccount.into())
+                .about("Configure confidential transfers for token account")
+                .arg(
+                    Arg::with_name("account")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_ACCOUNT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The address of the token account to configure confidential transfers for")
+                )
```

Comment:
For all of these, similar to `create-account`, how about taking in a `"mint"` and using the owner's ATA for that mint if `"account"` isn't specified?

Reply:
Yeah, that is a good idea. It turned out it is slightly clunky since `account` and `token` can all be optional. Please take a look. I think this suggestion also applies for all other commands that take in `account` as a required argument.